chart/templates/globalrole.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: management.cattle.io/v3
+builtin: false
+description: "Elemental Administrator Role"
+displayName: Elemental Administrator
+kind: GlobalRole
+metadata:
+  labels:
+    cattle.io/creator: norman
```

What does this label mean? what kind of information are we trying to record using it?

---

It is saying that this role was created by Norman https://github.com/rancher/norman
All Roles created in Rancher UI are getting this label. I think that it is good idea to keep it.

---

Should we change it? it will be easier to find out which part of the rancher owns it.
```suggestion
    cattle.io/creator: elemental
```